.github/workflows/fossa.yml

```diff
@@ -0,0 +1,27 @@
+name: FOSSA
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-go@v2
+        with:
+          go-version: "^1.13.1"
+      - run: go version
+      # Runs a set of commands to initialize and analyze with FOSSA
+      - name: run FOSSA analysis
+        env:
+          FOSSA_API_KEY: '304657e2357ba57b416b94e6b119131b'
```

Comment:
What is the origin of this key? Please add a comment describing it.

Reply:
@yurishkuro it's a FOSSA's push-only API token - https://docs.fossa.com/docs/api-reference#push-only-api-token; which is safe to be exposed publicly.

Reply:
but where does it come from? Is this ID specific to Jaeger project?

Reply:
Yes, it was created for Jaeger only.